integreat_cms/summ_ai_api/summ_ai_api_client.py

```diff
@@ -166,7 +175,12 @@ async def translate_text_fields(
                 # we need a NEW coroutine object.
                 # For that case, we a representation of our function which can be
                 # evaluated when needed, giving a new coroutine for the task each time.
-                partial(self.translate_text_field, session, text_field)
+                partial(
+                    self.translate_text_field,
+                    session,
+                    text_field,
+                    output_language_level,
+                )
```

Comment:
```suggestion
                partial(self.translate_text_field, session, text_field)
```
Then this can be reverted as well